Dungeoneer/src/com/interrupt/dungeoneer/gfx/GlRenderer.java

```diff
@@ -667,9 +666,8 @@ private void drawUI() {
 		uiBatch.setColor(Color.WHITE);
 
 		if(OverlayManager.instance.current() == null || !OverlayManager.instance.current().catchInput) {
-			// show crosshair if a ranged weapon is held
-			Item held = game.player.GetHeldItem();
-			if (held != null && (held.itemType == ItemType.bow || held.itemType == ItemType.junk || held.itemType == ItemType.wand)) {
+			// Draw crosshair
```

Comment:
Comments can be a smell for some refactoring.

How about you move this whole section into a new method `drawCrosshair`?

You probably then need to rename `showCrosshair` to something different, maybe `shouldDrawCrosshair`?

Reply:
I like this refactor. Done.